### PR TITLE
Update Docker image for Silverpeas version 6.2.2

### PR DESCRIPTION
**Pull request by mmoqui**

### CODE REVIEW

**Comment by mmoqui** [comment]
The naughty test fails because it doesn't find the tag 6.1.3 and 6.2.2 from refs/heads/master. It is normal because in our own process each stable version of Silverpeas comes from its own branch (and hence not from master). For example, the tag 6.1.3 comes from the branch 6.1.x and 6.2.2 comes from the branch 6.2.x. So that we can then release a patch version of Silverpeas for each minor version. 
Previously, this worked fine. But it seams you have changed some rules that break our process to maintain a different lifecycle for each minor versions of Silverpeas in their own branch.

Do you have a suggestion here how we can fix it without breaking our own process?

File: library/silverpeas [hunk 1]
```diff
@@ -2,8 +2,8 @@
 Maintainers: Miguel Moquillon <miguel.moquillon@silverpeas.org> (@mmoqui)
 GitRepo: https://github.com/Silverpeas/docker-silverpeas-prod.git
 
-Tags: 6.2.1, latest
-GitCommit: e97e55f0a21aae880a0054ef764cea0cb37e1e05
+Tags: 6.2.2, latest
+GitCommit: c6b68c650da8a95d9b258b4fe1f930fa9be737a6
```
Comment (yosifkit) [inline library/silverpeas:6]: Yeah, we added a test in https://github.com/docker-library/official-images/pull/11071 that ensures commits exist in the upstream. The fix is just telling it where to fetch the commits from. Adding a `GitFetch: ref/heads/6.2.x` line for the respective branch on each block should work.

Like this:
```suggestion
GitCommit: c6b68c650da8a95d9b258b4fe1f930fa9be737a6
GitFetch: refs/heads/6.2.x
```

Comment (mmoqui) [reply library/silverpeas:6]: Ok, thanks

**Comment by mmoqui** [comment]
> Are 6.1.x and 6.0.x still actively supported release lines?
> 
> Just as a reminder, removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them. (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Currently 6.1.x is actively supported. 6.0.x is indeed no more supported. We will remove it from the lib def at a next release of Silverpeas (normally in the month).

**Pull request merged**